### PR TITLE
Store and retrieve zenroom objects on fabric

### PR DESCRIPTION
**Pull request by albertolerda**

### CODE REVIEW

File: packages/fabric/src/index.ts [hunk 1]
```diff
@@ -152,7 +154,7 @@ export default async (req: Request, res: Response, next: NextFunction) => {
       if (zencode.match(QUERY)) {
         validateStep(FabricInterop.Contract); // The user must have set a contract
         const params = zencode.paramsOf(QUERY);
-        await submitAndRetry(params, "Could not ealuate transaction",
+        await submitAndRetry(params, "Could not ealuate transaction", 2,
```
Comment (puria) [inline packages/fabric/src/index.ts:157]: typo

File: packages/fabric/tests/index.ts [hunk 2]
```diff
@@ -23,6 +23,20 @@ test.serial("Include fabric middleware but do not use fabric statements", async
...
+});
+
+test("Store data", async (t) => {
+  const { app } = t.context;
+  var res = await app.post("/fabric_store");
+  t.is(res.status, 200, res.text);
+  t.is(typeof res.body.hash, "string");
+});
+
 test("Missing step", async (t) => {
```
Comment (puria) [inline packages/fabric/tests/index.ts:40]: Could you also please add a test in which there are many retrieve sentences? Does that case exists? Like a contract with:
```gherkin
Given I read from 'X' the data in tag 'aaaabbb999' and save the output into 'mimmo'
Given I read from 'Y' the data in tag 'cccccc' and save the output into 'mimmo'
```
or if can not be, then raise an error... even if I guess should be possible... anyway I need some testing cause will gonna change how the hooks works and don't want to break your code after the refactoring ;p

**Review by puria — CHANGES_REQUESTED** [review]

File: packages/fabric/tests/index.ts (hunk 2 above)
Comment (albertolerda) [reply packages/fabric/tests/index.ts:40]: Well for the moment only fabric is implemented with this kind of statement... So I cannot test two different statements with X different from Y...

I can surely test multiple statement which save data in different variables
```
Given I read from 'fabric' the data in tag 'aaaabbb999' and save the output into 'mimmo'
Given I read from 'fabric' the data in tag 'cccccc' and save the output into 'pippo'
```

What is the expected behaviour for saving multiple data in the same variable? I can save the names of all the tags and check that they are all different...

Comment (puria) [reply packages/fabric/tests/index.ts:40]: Also actually in different variables ;) like 'mimmo' and 'pippo'

**Pull request merged**